appliancevm/appliancevm/daemon.py

```
@@ -6,12 +8,12 @@
 log.configure_log('/var/log/zstack/zstack-appliancevm.log')
 logger = log.get_logger(__name__)
 
-import appliancevm
+from . import appliancevm
 
 def main():
     usage = 'usage: python -c "from appliancevm import daemon; daemon.main()" start|stop|restart'
     if len(sys.argv) != 2 or not sys.argv[1] in ['start', 'stop', 'restart']:
```

Ruff (0.15.10)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)